src/lib/select/select.html

```diff
@@ -1 +1,15 @@
-I'm a select!
\ No newline at end of file
+<div class="md-select-trigger" overlay-origin (click)="toggle()" #origin="overlayOrigin" #trigger>
+  <span class="md-select-placeholder" [@transformPlaceholder]="_getPlaceholderState()"> {{ placeholder }} </span>
```

Comment:
No, not directly. Since it's using CSS transitions and not animations, it's hard to share. There wasn't much left over to copy after that. In the next PR (for forms), I can reuse more of the input CSS (for things like the underline, etc).